setup.py

```diff
@@ -5,15 +5,13 @@
     version="1.0.0",
     packages=find_packages(),
     package_data={"revpi_provisioning": ["devices/*.yaml"]},
-    install_requires=[
-        "schema"
-    ],
+    install_requires=["schema"],
```

Comment:
```suggestion
    install_requires=[
        "schema",
    ],
```
Would Black accept that, with the comma at the end?

Reply:
Yes. See updated PR